taxi/tests/test_forms.py

```diff
@@ -0,0 +1,40 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from taxi.forms import DriverCreationForm, DriverLicenseUpdateForm
+
+
+class DriverCreationFormTest(TestCase):
+    def test_driver_creation_form_valid(self):
+        form_data = {
+            "username": "driver",
+            "password1": "lkjhfdsa",
+            "password2": "lkjhfdsa",
+            "first_name": "Driver",
+            "last_name": "Driverio",
+            "license_number": "OIU29032",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)
+
+    def test_driver_creation_form_not_valid(self):
+        form_data = {
+            "username": "driver",
+            "password1": "lkjhfdsa",
+            "password2": "lkjhfdsa",
+            "license_number": "",
+            "first_name": "",
+            "last_name": "",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertFalse(form.is_valid())
+
+
+class DriverLicenseUpdateFormTest(TestCase):

```

Comment:
did u test form validation here?

Reply:
yes and no, I test my function for license validation

Reply:
i mean, u are checking only valid case